example/setting_example.json

```diff
@@ -1,7 +1,8 @@
 {
   "client_id": "<your client id here>",
   "client_secret": "<your client secret here>",
+  "port": 3001,
   "url":  {
-    "redirect": "http://localhost/connect/res"
+    "redirect": "http://localhost:3001/connect/res"
```

Comment:
I like the idea but I think we should keep this file as simple as possible. Also maybe we can use process var https://nodejs.org/docs/latest/api/process.html#process_process_argv. It's very common and useful.